frontend/templates/product-list/sections/FilterableProductsSection/Toolbar.tsx

```diff
@@ -92,15 +96,27 @@ export function Toolbar(props: ToolbarProps) {
                      data-testid="list-view-button"
                      isActive={viewType === ProductViewType.List}
                      onClick={() => onViewTypeChange(ProductViewType.List)}
+                     borderColor="gray.300"
                   />
                   <ProductViewGridButton
                      aria-label="Select grid view"
                      data-testid="grid-view-button"
                      isActive={viewType === ProductViewType.Grid}
                      onClick={() => onViewTypeChange(ProductViewType.Grid)}
+                     borderColor="gray.300"
                   />
                </ProductViewSwitch>
             </Flex>
+
+            {currentRefinements.items.length > 0 && (
+               <Flex
+                  wrap="wrap"
+                  align="flex-start"
+                  display={{ base: 'flex', md: 'none' }}
+               >
+                  <CurrentRefinements />
```

Comment:
I noticed that the design displayed the current refinements in a single row with overflow-x hidden, whereas the implementation wrapped them onto multiple rows. It looks nice as implemented, but thought I'd point out the discrepancy.

| Spec | Implementation |
|-|-|
| ![image](https://user-images.githubusercontent.com/52104630/220207117-179720e4-c908-434f-8731-8812c5d63f10.png) | ![image](https://user-images.githubusercontent.com/52104630/220207068-5b5a31b5-91f4-4f5b-b686-689e8a28ab78.png) |

Reply:
Yep, you are right. During the development together with @mmarcon91 we discussed this.
Having chips on a single row make more room for the product list but make the current selection harder to read.
So we decided to keep the multiple row approach for a while and eventually to move to single row only if needed